### ISSUE TITLE
Conflict with outdated packages versions

### ISSUE THREAD

**Issue by CatHood0** [singerdmx/flutter-quill#2034]
### Is there an existing issue for this?

- [X] I have searched the [existing issues](https://github.com/singerdmx/flutter-quill/issues)

### Flutter Quill version

10.0.0

### Steps to reproduce

Just check the packages versions of the main pubspec.yaml of the repo. By example: `dart_quill_delta` is still at the 9.3.3 and if i want to add a dependency that depends on `dart_quill_delta` 10.0.0 will cause conflicts with `flutter_quill` package. 

Even you modify `dart_quill_delta` to the last version, then `flutter_quill_test` will also cause conflicts since it's depends on `flutter_quill` 9.0.0-dev (i don't remenber exactly the version but is too old)

### Code sample

<details><summary>Code sample</summary>

```dart
[Paste your code here]
```

</details>

### Screenshots or Video

<details>
<summary>Screenshots / Video demonstration</summary>

[Upload media here]

</details>

### Logs

<details><summary>Logs</summary>

```console
Because no versions of flutter_quill match >10.0.0 <11.0.0 and flutter_quill 10.0.0 depends on dart_quill_delta ^9.3.3, flutter_quill ^10.0.0 requires
  dart_quill_delta ^9.3.3.
And because every version of flutter_quill_delta_easy_parser depends on dart_quill_delta ^10.0.0, flutter_quill ^10.0.0 is incompatible with
  flutter_quill_delta_easy_parser.
```

</details>

**Comment by EchoEllet** [comment]
Should be fixed in [10.0.1](https://github.com/singerdmx/flutter-quill/releases/tag/v10.0.1).

The following packages are not yet compatible:

- [flutter_quill_to_pdf](https://pub.dev/packages/flutter_quill_to_pdf)
- [markdown_quill](https://pub.dev/packages/markdown_quill)
- [super_editor_quill](https://pub.dev/packages/super_editor_quill)

Updating to `10.0.0` is minimal and doesn't require changing the code.

**Comment by CatHood0** [comment]
> The following packages are not yet compatible

By now i update for [flutter_quill_to_pdf 2.0.0](https://github.com/CatHood0/flutter_quill_to_pdf/releases/tag/V-2.0.0) to fix outdated dependencies and perfomance issues
